integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -1227,6 +1227,138 @@ msgstr "Monatlich"
 msgid "Yearly"
 msgstr "Jährlich"
 
+#: cms/constants/language_color.py
+msgid "Mellow apricot"
+msgstr "Milde Aprikose"
+
+#: cms/constants/language_color.py cms/constants/poicategory.py
+msgid "Forest green"
+msgstr "Waldgrün"
+
+#: cms/constants/language_color.py
+msgid "Rose"
+msgstr "Rosa"
+
+#: cms/constants/language_color.py
+msgid "Tropical violet"
+msgstr "Tropisches Veilchen"
+
+#: cms/constants/language_color.py cms/constants/poicategory.py
+msgid "Red"
+msgstr "Rot"
+
+#: cms/constants/language_color.py cms/constants/poicategory.py
+msgid "Orange"
+msgstr "Orange"
+
+#: cms/constants/language_color.py cms/constants/poicategory.py
+msgid "Dark blue"
+msgstr "Dunkelblau"
+
+#: cms/constants/language_color.py
+msgid "Green blue"
+msgstr "Grün-blau"
```

Comment:
```suggestion
msgstr "Grünblau"
```
[Duden](https://www.duden.de/suchen/dudenonline/Gr%C3%BCn-blau) says no dash in between :)